### ISSUE TITLE
Accessibility improvements all over

### ISSUE THREAD

**Issue by D4RKONION** [D4RKONION/FAT#25]
Need to make the app much better for screen readers. At the very least, characters should be announced and tabbable, and the table should be given arias that read out each cell.

**Comment by D4RKONION** [comment]
Added in 3e9ada07a3312ff26ef9a54efb0f1e728d643642 and ed654ef162c1282210bf05c53fff057c6958fbf0. There's probably more that could be done but I'll wait til I get feedback on it
